src/content/structured/components/tree-view/guidance.mdx

```diff
@@ -67,3 +74,71 @@ For more information on canary components, read our approach to [releases and ve
     </IcTreeView>
   </div>
 </ComponentPreview>
+
+## When to use
+
+Use tree view to display further navigation options in the application, such as web pages following the home page.
+
+<DoubleDoDontCaution>
+  <DoDontCaution
+    imageSrc={treeViewFig1}
+    imageAlt="An example of using tree view to navigate throughout an app with nested pages."
+    state="good"
+    caption="Use tree view to navigate throughout hierarchical content."
+  />
+</DoubleDoDontCaution>
+
+Use tree view for hierarchical navigation options, such as related subsections to the current page or topic.
+
+Use tree view title bar to provide further clarity on the context of your tree view items.
+
+<DoubleDoDontCaution>
+  <DoDontCaution
+    imageSrc={treeViewFig2}
+    imageAlt="An example of using tree view with a title bar to display related content."
+    state="good"
+    caption="Use tree view with icons and a title bar to provide context to your data."
+  />
+</DoubleDoDontCaution>
+
+## When not to use
+
+Avoid using tree view for dense or complicated data display. Consider if this could be displayed better with a [data table](/components/data-table) or a grid.
+
+<DoubleDoDontCaution>
+  <DoDontCaution
+    imageSrc={treeViewFig3}
+    imageAlt="An example of using tree view for dense data with several data types."
+    state="bad"
+    caption="Don't use tree view for data that would benefit from being aligned vertically."
+  />
+</DoubleDoDontCaution>
+
+Avoid using tree view in a drawer/section with restricted width, as the nested layers for child tree items require width to expand into.
+
+Avoid using multiple tree views on the same web page, consider using tabs to separate sections.
```

Comment:
```suggestion
Avoid using multiple tree views on the same web page; consider using tabs which navigate to separate sections.
```
I think this might make the sentence a bit clearer?